Update ticketing stubs (#84535)
ansible/ansible: hacking/ticket_stubs/wider_discussion.md

Hi!

Thanks very much for your submission to Ansible. It means a lot to us.

We are interested in this idea and would like to see a wider discussion on it on one of our lists.
Reasons for this include:

* INSERT REASONS!

Because this project is very active, we're unlikely to see comments made on closed tickets and we lock them after some time.

> Can you please post an Ansible Forum so we can talk about this idea with the wider group?

Reviewer: ```suggestion
Please post your idea on the Ansible Forum so we can discuss it with the wider group.
```

* [Ansible Core on the Ansible Forum](https://forum.ansible.com/tag/ansible-core)

> * Matrix: [#devel:ansible.im](https://matrix.to/#/#devel:ansible.im)

Reviewer: I'd be strongly in favor of not recommending Matrix for this ticket stub. This is going to reduce the amount of people who can see the conversation, not widen it. Github and the forum are accessible to anyone, an account it only required to post, so I'd even prefer directing to ansible/proposals over Matrix...

```suggestion

```
